src/core/ipfs/libp2p.js

```diff
@@ -20,7 +20,11 @@ module.exports = function libp2p (self) {
       })
     },
     stop: (callback) => {
-      self._libp2pNode.swarm.close(callback)
+      try {
+        self._libp2pNode.swarm.close(callback)
+      } catch (err) {
+        console.log('It is fine :)')
```

Comment:
Let's remove the `try catch`, rebase master onto this branch, now it is really fine :)